client/src/entrypoints/contrib/typed_table_block/typed_table_block.js

```diff
@@ -141,6 +141,7 @@ export class TypedTableBlock {
   showAddColumnMenu(baseElement, callback) {
     this.addColumnMenuBaseElement = baseElement;
     baseElement.after(this.addColumnMenu);
+    baseElement.attr('aria-expanded', 'true');
```

Comment:
Presumably there ought to be a corresponding `baseElement.attr('aria-expanded', 'false');` on `hideAddColumnMenu`.

I think that when I originally wrote this code and chose the name `baseElement`, I didn't necessarily expect that to refer to the trigger button element - I think it was just the element that the menu dropdown was positioned relative to. Now that this change is effectively setting in stone that it is indeed the button, this is probably a good time to rename it accordingly - `baseElement` -> `triggerElement` and `addColumnMenuBaseElement` -> `addColumnMenuTrigger`?